Basic FE project template/index.html

```diff
@@ -5,16 +5,85 @@
     <meta name="viewport" content="width=device-width, initial-scale=1.0" />
 
     <link rel="stylesheet" type="text/css" href="ui/css/main.css" />
+    <link rel="stylesheet" href="ui-src/styles/queries.scss" />
+    <link rel="preconnect" href="https://fonts.googleapis.com" />
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Inter:wght@400;500;600;700&family=Syne:wght@400;500;600;700&display=swap"
+      rel="stylesheet"
+    />
 
-    <title>Website title</title>
+    <title>Car Saleing</title>
   </head>
 
   <body>
     <main>
-      <div>pateu gras</div>
-      <div>
-        <img src="ui-src/assets/images/audicar.webp" alt="car" />
-      </div>
+      <section class="why-beleasing">
+        <div class="container">
+          <div class="why-beleasing__container">
+            <div class="why-beleasing__title">
+              <div>
+                <h1>WHY BELEASING?</h1>
+              </div>
+              <div class="why-beleasing__description">
+                <p class="description">
+                  Nullam suspendisse mauris blandit cum pharetra odio mauris
+                  lacus risus. Velit at nam venenatis nunc lorem aliquam.
+                </p>
+              </div>
+              <ul class="advantages">
+                <li class="advantages__item">
+                  <div class="home--icon1">
```

Comment:
change the class names to home-icon1, home-icon2 and home-icon3, and don't forget to change the names in the scss files as well if needed.